CONTRIBUTING.md

```diff
@@ -11,4 +13,48 @@ Although contributing to the project is allowed and even encouraged it has to fo
 
 
 
-This is a *guideline* which means that some exceptions may be made. However it is recommend that you respect it for your contribution to be validated. If you have any other question or think a point is unclear or too limiting feel free to contact an @admin
\ No newline at end of file
+This is a *guideline* which means that some exceptions may be made. However it is recommend that you respect it for your contribution to be validated. If you have any other question or think a point is unclear or too limiting feel free to contact an @admin
+
+## Installing an editing
+
+### Requirements
+
+You will need python 3 and the library `discord.py`.
+
+*(We would also recommend to work on Linux :penguin:)*
+
+### The serious stuff
+
+Fork the repository from the [GitHub page](https://github.com/s0lst1ce/ForeBot). This will make a copy of the repo on your account.
+
+Then, on your computer, go to the directory where you want to install the bot and do this command:
+
+`git clone https://github.com/s0lst1ce/ForeBot.git`
+
+Then:
+
+`cd ForeBot` 
+
```

Comment:
Maybe add a sum up instruction first so that people know that they only need to clone the repo. After all some prefer to use GUI git apps.

Reply:
How would you do that ?

Reply:
Well only say that they need to clone the repo and say they can do it through their favorite Git app or directly using the git CLI. Then just leave all the good info you've written.